src/Microsoft.SqlTools.ServiceLayer/TableDesigner/DesignerPathUtils.cs

```diff
@@ -14,17 +14,19 @@ public static class DesignerPathUtils
         ///<summary>
         /// validate the path in the table designer change information.
         /// Below are the 3 scenarios and their expected path.
-        /// Note: 'index-{x}' in the description below are numbers represent the index of the object in the list.
+        /// Note: 'index{x}' in the description below are numbers represent the index of the object in the list.
 		/// 1. 'Add' scenario
-		///     a. ['propertyName1']. Example: add a column to the columns property: ['columns'].
-		///     b. ['propertyName1',index-1,'propertyName2']. Example: add a column mapping to the first foreign key: ['foreignKeys',0,'mappings'].
+		///     a. ['propertyName1',index1]. Example: add a column to the columns property: ['columns',0].
+		///     b. ['propertyName1',index1,'propertyName2','index2']. Example: add a column mapping to the first foreign key: ['foreignKeys',0,'mappings',0].
 		/// 2. 'Update' scenario
 		///     a. ['propertyName1']. Example: update the name of the table: ['name'].
-		///     b. ['propertyName1',index-1,'propertyName2']. Example: update the name of a column: ['columns',0,'name'].
-		///     c. ['propertyName1',index-1,'propertyName2',index-2,'propertyName3']. Example: update the source column of an entry in a foreign key's column mapping table: ['foreignKeys',0,'mappings',0,'source'].
+		///     b. ['propertyName1',index1,'propertyName2']. Example: update the name of a column: ['columns',0,'name'].
+		///     c. ['propertyName1',index1,'propertyName2',index2,'propertyName3']. Example: update the source column of an entry in a foreign key's column mapping table: ['foreignKeys',0,'mappings',0,'source'].
 		/// 3. 'Remove' scenario
-        ///     a. ['propertyName1',index-1]. Example: remove a column from the columns property: ['columns',0'].
-		///     b. ['propertyName1',index-1,'propertyName2',index-2]. Example: remove a column mapping from a foreign key's column mapping table: ['foreignKeys',0,'mappings',0].
+        ///     a. ['propertyName1',index1]. Example: remove a column from the columns property: ['columns',0'].
+		///     b. ['propertyName1',index1,'propertyName2',index2]. Example: remove a column mapping from a foreign key's column mapping table: ['foreignKeys',0,'mappings',0].
+		/// 4. 'Move' scenario
+        ///     a. ['propertyName1',from,to]. Example: move the second column to the third place: ['columns',1,2].
```

Comment:
just realized I missed this one in my first review, here the path should be the same as remove:
'propertyName1',index1
'propertyName1',index1,'propertyName2',index2

and the value will be the to Index.

Reply:
![image](https://user-images.githubusercontent.com/13777222/165385539-b7db757c-396e-4377-b2bf-ba8f58be4128.png)